.github/workflows/build.yml

```diff
@@ -0,0 +1,163 @@
+name: Build and Test
+
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: "*"
+
+jobs:
+  build:
+    name: Build tljh_repo2docker
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Base Setup
+        uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
+
+      - name: Lint the application
+        shell: bash -l {0}
```

Comment:
Maybe this `shell` (and the others below) could be removed if we use the `base-setup` action?